package.json

```diff
@@ -33,6 +37,12 @@
     "webpack-cli": "^3.1.1"
   },
   "dependencies": {
+    "body-parser": "^1.18.3",
```

Comment:
Where did that came from?
Shouldn't be a devdependency?

Reply:
That comes from server.js
[eslint] 'body-parser' should be listed in the project's dependencies, not devDependencies.

Reply:
I would still try to fix that. You have it installed - things from dependencies will be used on frontend https://stackoverflow.com/questions/44939304/eslint-should-be-listed-in-the-projects-dependencies-not-devdependencies

Reply:
As below.